app/services/encryption.py

```
@@ -0,0 +1,139 @@
+    try:
+        from cryptography.hazmat.primitives.ciphers.aead import AESGCM
+
+        nonce = os.urandom(_NONCE_SIZE)
+        aesgcm = AESGCM(key)
+        ciphertext = aesgcm.encrypt(nonce, plaintext.encode("utf-8"), None)
+
+        # Store as: nonce (12 bytes) + ciphertext+tag
+        encrypted_blob = nonce + ciphertext
+        return base64.urlsafe_b64encode(encrypted_blob).decode("utf-8")
+
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)